docs/filesystem_layer/stratum1.md

```diff
@@ -50,8 +50,16 @@ Then install Ansible roles for EESSI:
 ansible-galaxy role install -r requirements.yml -p ./roles --force
 ```
 
+Make sure you have enough space in `/srv` (on the Stratum 1) since the snapshot of the Stratum 0
+will end up there by default. If you want to alter the path where the snapshot gets copied to you
+can edit this variable in `roles/galaxyproject.cvmfs/defaults/main.yml`:
```

Comment:
Better not to edit the files from that role, but you can override this variable per host in the `inventory/host_vars` folder (or for all your hosts in `inventory/local_site_specific_vars.yml`, or per group in `inventory/group_vars`, but I guess you often want to do this per host).

Reply:
Thanks for clarifying!